cmd/statreceiver/config.lua

```diff
@@ -3,24 +3,30 @@
 --  * filein(path)
 -- multiple sources can be handled in the same run (including multiple sources
 -- of the same type) by calling deliver more than once.
-source = udpin("localhost:9000")
+source = udpin(":9000")
 
 -- multiple metric destination types
 --  * graphite(address) goes to tcp with the graphite wire protocol
 --  * print() goes to stdout
 --  * db("sqlite3", path) goes to sqlite
 --  * db("postgres", connstring) goes to postgres
-graphite_out = graphite("localhost:5555")
+ graphite_out_local = graphite(graphite(os.getenv(GR_OUT_LOCAL)))
+ graphite_out_remote1 = graphite(graphite(os.getenv(GR_OUT_REMOTE1)))
+ graphite_out_remote2 = graphite(graphite(os.getenv(GR_OUT_REMOTE2)))
 db_out = mcopy(
   db("sqlite3", "db.db"),
-  db("postgres", "user=dbuser dbname=dbname"))
+  db("postgres", "host=os.getenv(PG_HOST) port=os.getenv(PG_PORT) user=os.getenv(PG_USER) password=(PG_PASS) dbname=os.getenv(PG_DBNAME) sslmode=disable"))
```

Comment:
I'm ok with this file staying an example. I envision we're going to have to change the config file often enough we'll just overwrite it with one stored in a configmap in kubernetes. I don't think there needs to be any changes done to this file.